mephisto/server/blueprints/acute_eval/webapp/src/components/core_components.jsx

```diff
@@ -79,18 +79,28 @@ function ChatMessage({ message, model, is_primary_speaker }) {
           backgroundColor: otherspeaker_color,
         }),
   };
-  return (
-    <div style={message_container_style}>
-      <div style={message_style}>{message}</div>
-    </div>
-  );
+  if (image_src !== null) {
+    return (
+      <div style={message_container_style}>
+        <div style={message_style}>
+          {message}<img src={image_src} width="320" alt='Image'/>
+        </div>
+      </div>
+    );
+  } else {
+    return (
+      <div style={message_container_style}>
+        <div style={message_style}>{message}</div>
+      </div>
+    );
+  }
 }
 
 function MessageList({ task_data, index }) {
   let messageList;
 
   if (task_data.pairing_dict === undefined) {
-    messageLists = (
+    messageList = (
```

Comment:
I assume this is supposed to be `messageList`? Because otherwise this variable is never used